app/controllers/better_together/notifications_controller.rb

```diff
@@ -35,6 +35,17 @@ def mark_as_read # rubocop:todo Metrics/AbcSize, Metrics/MethodLength
         end
       end
     end
-    # rubocop:enable Metrics/MethodLength
+
+    def mark_notification_as_read(id)
+      @notification = helpers.current_person.notifications.find(id)
+      @notification.update(read_at: Time.current)
+    end
+
+    def mark_message_as_read(id)
+      @notification = helpers.current_person.notifications.unread.includes(
```

Comment:
This variable name should be plural, as the method chain will return a collection of notifications for the record.